### PR TITLE
P0 alert button

### PR DESCRIPTION
**Pull request by anupkgurung**

### CODE REVIEW

File: css/buttons.css [hunk 1]
```diff
@@ -0,0 +1,75 @@
+.btn {
+    outline: none;
+    border: none;
```
Comment (Aditi0505) [inline css/buttons.css:3]: Suggestion: You can remove this as you are defining border again. The border value is none by default. Let me know about your thoughts on this.

File: css/typography.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}
+.link-style-none {
+    text-decoration: none;
+    color: inherit;
+    background-color: transparent;
+    border: none;
+    cursor: pointer;
+}
+  
+  /* .big-text{
```
Comment (Aditi0505) [inline css/typography.css:37]: Suggestion: Can remove commented out codes if not needed.

**Review by Aditi0505 — APPROVED** [review]

File: css/typography.css (hunk 2 above)
Comment (anupkgurung) [reply css/typography.css:37]: sure thanks

File: css/buttons.css (hunk 1 above)
Comment (anupkgurung) [reply css/buttons.css:3]: yeah will refactor all at once. Thanks !!

File: css/alert.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    display: flex;
+    justify-content: center;
+}
+.alert-container .alert{
+    width: 25rem;
+}
+.alert {
+    margin: .5rem;
+    padding: .5rem;
+    border-radius: 5px;
```
Comment (HeyNitin) [inline css/alert.css:14]: Can be in 'rem' for css consistency[ least priority ]

File: css/alert.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    background-color: var( --blue-bg);
+    color: var(--primary-blue);
+}
+
+/***** MEDIA QUERY *****/
+@media only screen and (max-width: 400px) {
+    .alert {   
+        font-size: 16px;
+        line-height: 24px;
+    }
```
Comment (HeyNitin) [inline css/alert.css:43]: can be converted through rem also [least priority]

**Review by HeyNitin — APPROVED** [review]
Aditi has suggested some changes, apart from them, Looks good to merge to me.

File: css/alert.css (hunk 4 above)
Comment (anupkgurung) [reply css/alert.css:43]: sure !!

**Pull request merged**